test/e2e/ten_1

```diff
@@ -0,0 +1,10 @@
+0
```

Comment:
FWIW -- please do not populate such simple to generate files within repo.  If needed -- a fixture with a session scope could create such files to be re-used across tests.